src/libraries/math/SafeMathUint256.sol

```diff
@@ -45,4 +44,22 @@ library SafeMathUint256 {
             return b;
         }
     }
+
+    function safeToMultiply(uint256 a, uint256 b) internal constant returns (bool) {
+        uint256 c = a * b;
+        if (a == 0 || c / a == b) {
+            return true;
+        }
+        return false;
+    }
+
+    // Float [fixed point] Operations
+    function fxpMul(uint256 a, uint256 b) internal constant returns (uint256) {
```

Comment:
Given the name (`fxpMul`) this should _probably_ include a base parameter.  I have argued against fixed point math in the past and lost, so I won't try to convince you to remove it.  ;)

At the moment, this implementation is 10^18 fixed point multiply, yet the name doesn't convey that it is 10^18 fixed point.  Recommend either changing the name to be more clear, or adding a base parameter.

Reply:
Ok, I agree that adding the base would make it clearer.  Maybe call it fxpMulBase18() instead?

Reply:
I'm not sure, the actual base is `10^18` (different from `2^18` base for example).  This is part of why I'm generally against fixed point math, because doing it well is hard.  :/  I'll leave it up to you to make the final call of leaving it as is, changing the name (and if so, what to change it to), or adding a base parameter.  Good luck!